src/runtime/runtime-debug.cc

```diff
@@ -1014,8 +1014,33 @@ RUNTIME_FUNCTION(Runtime_RecordReplayAssertExecutionProgress) {
     CHECK(gRecordReplayHasCheckpoint);
   }
 
-  recordreplay::Assert("ExecutionProgress %zu %s:%d:%d",
-                       *gProgressCounter, name.c_str(), info.line + 1, info.column);
+  recordreplay::Assert(
+      "JS ExecutionProgress PC=%zu scriptId=%d @%s:%d:%d (%d %d %d %d)",
+      *gProgressCounter, script->id(), name.c_str(), info.line + 1, info.column,
+      recordreplay::AreEventsDisallowed(),
+      !recordreplay::HasDivergedFromRecording(),
```

Comment:
It's pretty meaningless to include AreEventsDisallowed() and HasDivergedFromRecording() in an assertion because we'll emit a warning whenever asserting and events are disallowed, and will ignore assertions after diverging from the recording.  These should be removed.